pkg/realmd/operation.html

```diff
@@ -76,10 +76,10 @@ <h4 class="modal-title realms-op-title"></h4>
             </tr>
             <tr>
               <td colspan="3">
-                <div class="realms-op-error" hidden></div>
-                  <pre class="realms-op-diagnostics" hidden>
-                  </pre>
+                <div class="realms-op-error alert alert-danger" hidden>
```

Comment:
Removing the "alert" class here displays the error icon and message. The font styling looks bad, though.